src/main/java/reposense/model/RepoLocation.java

```diff
@@ -68,6 +69,7 @@ private void verifyLocation(String location) throws InvalidLocationException {
         isValidGitUrl = matcher.matches();
 
         if (!isValidPathLocation && !isValidGitUrl) {
+            ErrorSummary.getInstance().addErrorMessage(location, "Invalid location.");
```

Comment:
Would it be better to display the exact URL here? Sth like [URL] pointing to an invalid location

Reply:
Sure, no problem